VERSION

```diff
@@ -1 +1 @@
-1.1.0
\ No newline at end of file
+master
```

Comment:
Per @lranjbar , just leave as is. CI is already tagging image with master. Version file will eventually be removed.

Reply:
That is true, but "make docker" will still create "1.1.0".  Shouldn't we be consistent for local builds?

Reply:
That is only for building a local docker image. Really should be set to `dev` in the makefile, but all that is changing as DevOps executes their CI changes.

Reply:
Backed out the change to VERSION.